input/install.sh

```diff
@@ -437,6 +449,7 @@ Options:
 
 Versions:
   latest              The latest release
+  v1-latest           The latest release in the 1.x line
```

Comment:
I have the impression that if a user wants v1 they will specify a version (and that v1-latest is only useful for dev purposes).